dataactvalidator/interfaces/validatorValidationInterface.py

```diff
@@ -399,6 +403,18 @@ def getRuleSeverityByName(self, ruleSeverityName):
             "Multiple rule severities found with name {}".format(ruleSeverityName))
         return query
 
+    def getLongToShortColname(self):
+        """Return a dictionary that maps schema field names to shorter, machine-friendly versions."""
+        query = self.session.query(FileColumn.name, FileColumn.name_short)
+        dict = {row.name:row.name_short for row in query}
```

Comment:
Yes that line is clear, my problem is that the query hasn't had something like query.all() done, so it shouldn't have results available.  If sqlalchemy is being overly clever and allowing you to pull a result out of an unexecuted query by executing it, my concern would then become how often the query is executed if you're iterating through it.  Is there an advantage to doing it this way rather than having the previous line end with .all() to make it more explicit?